snackbar

CalinFlorinTUDOR opened the pull request.

CalinFlorinTUDOR committed:
snackbar

Hionel reviewed (changes requested):
Pentru a nu aglomera componenta de auth , ar trebuii snackbar-ul mutat intr-un service facut pentru el . In serviciul asta ar trb sa fie 2 functii una pt notificariile de success si una de erori care sa primeasca prin parametru mesajul aferent erorilor .

CalinFlorinTUDOR committed:
notifier.service.ts

Horia,trebuie totusi sa facem if-ul ala sa dea erroare in else si sa apara divul ala rosu si verde....

CalinFlorinTUDOR committed:
modificari
